build-support/ci_lists/unit_remote_blacklist.txt

```diff
@@ -0,0 +1 @@
+contrib/go/tests/python/pants_test/contrib/go/tasks
```

Comment:
?

Reply:
I added this on @Eric-Arellano 's suggestion because this one test was preventing CI from passing last week. It's still not passing CI though so it doesn't seem to have helped.